src/registrar/tests/test_views.py

```diff
@@ -1315,6 +1317,23 @@ def test_domain_authorizing_official_content(self):
         )
         self.assertContains(page, "Testy")
 
+    def test_domain_org_name_address(self):
+        """Can load domain's org name and mailing address page."""
+        page = self.client.get(
+            reverse("domain-org-name-address", kwargs={"pk": self.domain.id})
+        )
+        # once on the sidebar, once in the page title, once as H1
+        self.assertContains(page, "Organization name and mailing address", count=3)
+
+    def test_domain_org_name_address_content(self):
+        """Org name and address information appears on the page."""
+        self.domain_information.organization_name = "Town of Igorville"
+        self.domain_information.save()
+        page = self.app.get(
+            reverse("domain-org-name-address", kwargs={"pk": self.domain.id})
+        )
+        self.assertContains(page, "Town of Igorville")
+
```

Comment:
**suggestion**: I think that we could use another test for the form here. See `test_domain_user_add_form` for an example. Basically you want to load the page with `self.app`, fill out one of the form fields, click `submit` and then verify that the new value you entered is saved.